ios-unit-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: iOS Unit Tests
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
```

Comment:
Should we add a `paths` prop to only run this if there are changes to the iOS folder?

Reply:
I think we'd also want to run it if there's changes to the `shared/commonMain` and `shared/iosMain` folders?  And changes to gradle/other project settings could break it too...might be safer to say "any file _except_ `androidApp/*` and `shared/*`/`androidMain/*`"?

Reply:
Yeah. I've also found it useful to have them run if the workflow file itself changes

Reply:
@j-mateo updated paths, think those look ok?

Reply:
Just realized that we can `exclude` paths instead of having to include all of these.


```
paths-ignore:
      - "**.md"
      - androidApp
```
etc

Reply:
but we can do that in a follow-up pr if you want